src/Container/Lifetime/LifetimeContainer.cs

```diff
@@ -105,27 +103,14 @@ protected virtual void Dispose(bool disposing)
             {
                 if (_items.Count == 0)
                     disposables = EmptyDisposables;
+                else if (_items.Count < 20)
```

Comment:
New results after this commit:

  CreateChildContainer |         50   |   1.499 us | 0.0220 us | 0.0195 us |   1.00 |     0.00 |  0.1984 |     840 B |
 ChildDisposableObject |         50 |  49.773 us | 0.6795 us | 0.6024 us |  33.21 |     0.57 |  6.8970 |   28961 B |
  CreateChildContainer |        100  |   1.515 us | 0.0300 us | 0.0321 us |   1.00 |     0.00 |  0.1984 |     840 B |
 ChildDisposableObject |      100 | 103.548 us | 2.0075 us | 1.8779 us |  68.36 |     1.83 | 13.6719 |   57602 B |

Reply:
I am not sure what are you fixing here?